compiler/z/codegen/OMRInstOpCodeEnum.hpp

```diff
@@ -60,7 +60,6 @@
    XHHR,                // Exclusive OR High (High <- High)
    XHLR,                // Exclusive OR High (High <- Low)
    XLHR,                // Exclusive OR High (Low <- High)
-   XPCALLDESC,          // zOS-31 LE Call Descriptor.
```

Comment:
This used to be used for an "inline" XPLINK call descriptor. It is now replaced but a much more robust and documented `XPLINKCallDescriptorSnippet`